frontend-react/src/App.js

```diff
@@ -0,0 +1,70 @@
+
+import {useState} from 'react'
+import {Switch, Route, Redirect} from 'react-router-dom'
+
+import Header from './Header.js';
+import Main from './pages/Main.js';
+import About from './pages/About.js';
+import Settings from './pages/Settings.js';
+
+import './App.css';
+
+const baseUrl = "http://explorehsk.com:5000/"
```

Comment:
This is the kind of thing you can put in a .env file and/or in your github environment variables

Reply:
Great suggestion! Addressed in ba83712

I got more information about usage with `create-react-app` in https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env

Reply:
I am not familiar with github env variables. Can you explain how you would use those in this context?